lib/src/api/mod.rs

```diff
@@ -6,6 +6,7 @@ pub mod method;
 pub mod opt;
 
 mod conn;
+pub mod model;
```

Comment:
Would you mind dropping this module? The live query API PR introduces these types in a different place. In any case, these types are not needed for this particular PR.

Reply:
Code moved to dbs